src/rust/ide/src/view/node_searcher.rs

```diff
@@ -80,6 +86,28 @@ impl NodeSearcher {
         self.display_object.add_child(&self.text_field.display_object());
```

Comment:
docstring says `/// Show NodeSearcher if it is invisible.`
but I don't see in this function any "isn't this already visible?" checks